main.tf

```diff
@@ -72,6 +86,9 @@ locals {
 
   azure_easyauth_callback                 = "/.auth/login/aad/callback"
 
+  owner_group_members                     = data.azuread_group.owner_groups!= null ? flatten([for item in data.azuread_group.owner_groups: item.members]):[]
```

Comment:
I think we also need to filter members based on their type here so we don't accidentally try to add a nested group, which I believe would cause an error.

Reply:
Agreed.  Added a step to get the users from the  groups so non-users are filtered out.